www/src/components/shell/helpers.tsx

```diff
@@ -0,0 +1,5 @@
+function isAlphanumeric(value): boolean {
+  return (/^[a-zA-Z0-9-]+$/.test(value))
+}
+
+export { isAlphanumeric }
```

Comment:
Why exporting from there?

Reply:
It is just easier for me to see the actual module API that is being exposed to the outside and I am using the same pattern everywhere.

Reply:
Ok, just asking. For me it's easier to see if method is exported or not if it's written in the same line. In this case I also need to check end of the file.

Reply:
Not a big deal though.